it/connectors/data/Api1871Test.scala

```diff
@@ -39,5 +39,38 @@ trait Api1871Test {
       |}
       |""".stripMargin
 
+  // I really want some space where when we have some issue we can share with CSD, and then have people that come and answer on that solution. Not just waiting for all hands or Q&A sessions.
```

Comment:
@palequezal please look into the commented code, it is not the intended comment.